infrastructure/resource_group_level/modules/portal-api.bicep

```diff
@@ -0,0 +1,246 @@
+param location string
+param environment string
+param containerRegistryName string
+param appServicePlanEgressSubnetId string
+param appServicePlanId string
+param privateEndpointsSubnetId string
+param cosmosDbAccountName string
+param cosmosDbDatabaseName string
+param logAnalyticsWorkspaceId string
+param externalApiHostname string
+param dtfsCentralApiHostname string
+param storageAccountName string
+
+var dockerImageName = '${containerRegistryName}.azurecr.io/portal-api:${environment}'
+var dockerRegistryServerUsername = 'tfs${environment}'
+
+// These values are taken from GitHub secrets injected in the GHA Action
+@secure()
+param secureSettings object = {
+
+}
+
+// These values are taken from an export of Configuration on Dev (& validating with staging).
+@secure()
+param additionalSecureSettings object = {
+  DOCKER_REGISTRY_SERVER_PASSWORD: 'test-value'
+}
+
+// These values are taken from GitHub secrets injected in the GHA Action
+@secure()
+param secureConnectionStrings object = {
```

Comment:
It's a bit weird that these are sent in as connection strings. However, that's the way it was done in the original.